### ISSUE TITLE
missed mouse click

### ISSUE THREAD

**Issue by pputre** [zacfilan/brimstone-recorder#242]
There are times when a recorded mouse click does not seem execute and the screen comparison fails.  I manually re-execute the click and then the test goes on it's merry way.  Can we add an option for a mouse click retry attempt if the expected screen is significantly different after a failure (maybe use a set/configurable pixel minimum).  This appears mostly when I try to loop the STT's to run through x (3) times.

**Comment by patchworksapp** [comment]
Yep. Happy to. I think this is the case when there is an invisible overlay
on the screen which we put up to prevent the user from interacting with the
screen when the app is not stable. The user will just retry the action.
I'll add the option when I get back next week. I'm in the Cape this week.
